specification/ai/data-plane/OpenAI.Assistants/OpenApiV3/2024-02-15-preview/assistants_generated.yaml

```diff
@@ -36,25 +36,29 @@ paths:
             type: integer
             format: int32
             default: 20
+          explode: false
```

Comment:
this the the openpai3 bug fix where we mis interpreted the `explode` default value in openapi3 query params(explode: true) but in TypeSpec its always `explode: false` by default.